File: benchmarks/fixtures/completion/ts/service.ts
import { UserRepo as PrimaryRepo } from './repos/primary'

export class UserService {
  constructor(private readonly repo: PrimaryRepo) {}
  load(id: string) { return this.repo.findUser(id) }
}

export function loadWith(repo: PrimaryRepo, id: string) {
  const alias = repo
  return alias.findUser(id)
}

File: benchmarks/fixtures/completion/python/service.py
from repos.primary import UserRepo as PrimaryRepo

class UserService:
    def __init__(self, repo: PrimaryRepo):
        self.repo = repo

    def load(self, user_id: str) -> str:
        return self.repo.find_user(user_id)

def load_with(repo: PrimaryRepo, user_id: str) -> str:
    return repo.find_user(user_id)
